app/MMB/src/list/FlightList.js

```diff
@@ -63,51 +62,39 @@ export class FlightList extends React.Component<Props> {
   };
 
   render = () => {
-    const flights = idx(this.props.data, _ => _.allBookings.edges) || [];
-    return (
-      <ScrollView contentContainerStyle={styles.container}>
-        <Text style={styles.upcomingText}>
-          <Translation id="mmb.my_bookings.future_trips" />
-        </Text>
-        {flights.map(this.renderItem)}
-      </ScrollView>
-    );
+    const flights = idx(this.props.data, _ => _.edges) || [];
+
+    if (flights.length === 0) {
+      return null;
+    }
+
+    return <React.Fragment>{flights.map(this.renderItem)}</React.Fragment>;
```

Comment:
@tbergq Why fragment? Can I remove it?

Reply:
Yes, you are right, the fragment is abundant.